docs/wallets/cli/startup-basics.md

```diff
@@ -49,6 +49,6 @@ The above example would first look to the `dcrd` configuration file for settings
 [Full List of Options for Each Application](dcrd-and-dcrwallet-cli-arguments.md)
 
 
-## Next:
+## Next
```

Comment:
Looks good, just wanted to make sure you intended to put this edit in this PR, as it's dcrd not dcrlnd.

Reply:
Yep confirm this is deliberate. After [this review](https://github.com/decred/dcrdocs/pull/1055#discussion_r372317604), I ran a grep for headings which were ending with a colon and this one came up.